docs/getting-started/edfi-exchange/technology/data-import-templates.md

```diff
@@ -0,0 +1,22 @@
+# Data Import Templates
+
+![Ed-Fi Logo](https://edfidocs.blob.core.windows.net/$web/img/edfi-exchange/technology/image.png)
+
+## Description
+
+Data Import allows the publishing and consuming of templates of other Data Import users. This contribution lists currently published and available templates.
+
+## Download
+
+* **Code & Documentation:** [Published Data Import Templates](https://edfi.atlassian.net/wiki/spaces/EDFITOOLS/pages/24119005/Published+Data+Import+Templates)
```

Comment:
This page should be updated this week, but we'll probably complete this  DOC-148 PR first, so I suppose we should just leave the old URL alone for now. We'll catch it later when we do a sweep through to find other leftover Confluence links.